kubernetes/helm/values.override_production.auction-ui.k8s.sidestream.tech.yaml

```diff
@@ -0,0 +1,11 @@
+# Value overrides for the deployment to the following host:
+host: "production.auction-ui.k8s.sidestream.tech"
```

Comment:
But we should set several hosts over here, including:
- `auctions.makerdao.network`
- `unified-auctions.makerdao.com`

So the DNS switching can be done with no downtime

CC @florianstoecker: we should whitelist another origin for the production frontend infura key

Reply:
which one? Currently Whitelisted are:
- https://unified-auctions.makerdao.com
- https://production.auction-ui.k8s.sidestream.tech
- https://auctions.makerdao.network/

Reply:
This list is enough, thank you (: